### PR TITLE
docs: add multiple examples

### PR DESCRIPTION
**Pull request by afiune**
## Summary

We are adding a few examples to clarify how to onboard multiple scenarios:
* single tenant single subscription
* single tenant multiple subscriptions
* single tenant all subscriptions
* multiple tenants all subscriptions

## How did you test this change?

Added these examples to run in our CI pipeline.

## Issue
https://lacework.atlassian.net/browse/GROW-1446

### CODE REVIEW

**Review by jon-stewart — APPROVED** [review]
Do we need an example for subscription_id in the azurerm block (GROW-1444)?

File: examples/default-activity-log/README.md [hunk 1]
```diff
@@ -17,4 +18,4 @@ module "az_activity_log" {
 }
 ```
 
-For detailed information on integrating Lacework with Azure see [Azure Compliance & Activity Log Integrations - Terraform From Any Supported Host](https://support.lacework.com/hc/en-us/articles/360058966313-Azure-Compliance-Activity-Log-Integrations-Terraform-From-Any-Supported-Host)
+For detailed information on integrating Lacework with Azure see [Azure Compliance & Activity Log Integrations - Terraform From Any Supported Host](https://docs.lacework.com/onboarding/azure-compliance-and-activity-log-integrations-terraform-from-any-supported-host)
```
Comment (sthetland) [inline examples/default-activity-log/README.md:21]:
```suggestion
For detailed information on integrating Lacework with Azure, see [Azure Compliance & Activity Log Integrations - Terraform From Any Supported Host](https://docs.lacework.com/onboarding/azure-compliance-and-activity-log-integrations-terraform-from-any-supported-host).
```

File: examples/existing-storage-account/README.md [hunk 2]
```diff
@@ -19,4 +20,4 @@ module "az_activity_log" {
 }
 ```
 
-For detailed information on integrating Lacework with Azure see [Azure Compliance & Activity Log Integrations - Terraform From Any Supported Host](https://support.lacework.com/hc/en-us/articles/360058966313-Azure-Compliance-Activity-Log-Integrations-Terraform-From-Any-Supported-Host)
+For detailed information on integrating Lacework with Azure see [Azure Compliance & Activity Log Integrations - Terraform From Any Supported Host](https://docs.lacework.com/onboarding/azure-compliance-and-activity-log-integrations-terraform-from-any-supported-host)
```
Comment (sthetland) [inline examples/existing-storage-account/README.md:23]:
```suggestion
For detailed information on integrating Lacework with Azure, see [Azure Compliance & Activity Log Integrations - Terraform From Any Supported Host](https://docs.lacework.com/onboarding/azure-compliance-and-activity-log-integrations-terraform-from-any-supported-host).
```

File: examples/multiple-subscriptions-in-tenant/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Integrate Multiple Azure Subscriptions within a Tenant
+
+The following example integrates multiple Azure subscriptions within a
+single Azure tenant.
```
Comment (sthetland) [inline examples/multiple-subscriptions-in-tenant/README.md:4]: as above..

```suggestion
single Azure tenant with Lacework for Activity Log analysis.
```

File: examples/multiple-subscriptions-in-tenant/README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  subscription_ids = [
+    "00000000-0000-0000-0000-000000000000",
+    "00000000-0000-0000-0000-000000000001",
+    "00000000-0000-0000-0000-000000000002",
+    "00000000-0000-0000-0000-000000000003"
+  ]
+}
+```
+
+For detailed information on integrating Lacework with Azure see [Azure Compliance & Activity Log Integrations - Terraform From Any Supported Host](https://docs.lacework.com/onboarding/azure-compliance-and-activity-log-integrations-terraform-from-any-supported-host)
```
Comment (sthetland) [inline examples/multiple-subscriptions-in-tenant/README.md:26]:
```suggestion
For detailed information on integrating Lacework with Azure, see [Azure Compliance & Activity Log Integrations - Terraform From Any Supported Host](https://docs.lacework.com/onboarding/azure-compliance-and-activity-log-integrations-terraform-from-any-supported-host).
```

File: examples/multiple-tenants/README.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
+# Integrate Multiple Azure Tenants
+
+The following example integrates multiple Azure tenants and all their subscriptions
```
Comment (sthetland) [inline examples/multiple-tenants/README.md:3]:
```suggestion
The following example integrates multiple Azure tenants and all their subscriptions with Lacework for Activity Log analysis.
```

File: examples/multiple-tenants/README.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+  providers = {
+    azurerm = azurerm.tenant_2
+  }
+
+  all_subscriptions = true
+}
+```
+
+For detailed information on integrating Lacework with Azure see [Azure Compliance & Activity Log Integrations - Terraform From Any Supported Host](https://docs.lacework.com/onboarding/azure-compliance-and-activity-log-integrations-terraform-from-any-supported-host)
```
Comment (sthetland) [inline examples/multiple-tenants/README.md:43]:
```suggestion
For detailed information on integrating Lacework with Azure. see [Azure Compliance & Activity Log Integrations - Terraform From Any Supported Host](https://docs.lacework.com/onboarding/azure-compliance-and-activity-log-integrations-terraform-from-any-supported-host).
```

**Review by sthetland — CHANGES_REQUESTED** [review]
Looks good basically.. some minor comments/suggestions.

**Review by sthetland — APPROVED** [review]
LGTM

**Pull request merged**